keyboards/dztech/dz65rgb/readme.md

```diff
@@ -1,10 +1,17 @@
 # DZ65RGB
 
+
+
```

Comment:
Leaving space for a photo later?

Reply:
Oops, I added that to compare between the four readmes in their separate tabs 😄